main.js

```diff
@@ -160,4 +160,212 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+
+// 1a:
+// 1) create function
+function countLeftOverStock() {
+  let sum = 0;
+// 2) create new array with leftover stock per tv
+  const leftOverStock = inventory.map((item) => {
+    return item.originalStock - item.sold;
+  });
+// 3) add together all indexes of the new array
+  for (let i = 0; i < leftOverStock.length; i++) {
+    sum += leftOverStock[i];
+  }
```

Comment:
Hier gebruik je de for loop na een map methode, waardoor je map methode eigenlijk overbodig raakt.